docs/reference/configuration.rst

```diff
@@ -127,3 +127,11 @@ Client connections
 :eql:synopsis:`query_execution_timeout -> std::duration`
   How long an individual query can run before being aborted. A value of
   ``<duration>'0'`` disables the mechanism; it is disabled by default.
+
+.. note::
+
+    For ``session_idle_transaction_timeout`` and ``query_execution_timeout``,
+    values under 1ms are rounded down to zero, which will disable the timeout.
+    In order to set a timeout, please set a duration of 1ms or greater.
+
+    ``session_idle_timeout`` can take values below 1ms.
```

Comment:
I could see missing this note if I was only looking for `session_idle_transaction_timeout` documentation. Its repetitive but maybe this should just be part of the documentation and duplicated for each config value?

Reply:
Yeah, I like that idea.